translations/src/config.js

```diff
@@ -34,20 +34,18 @@ class Config {
     tr: { rtl: false },
     pl: { rtl: false },
     ti: { rtl: false },
-    ku: {
-      rtl: false,
+    'ckb-Arab': {
+      rtl: true,
       additionalFont: 'lateef'
     },
+    'ckb-Latn': { rtl: false },
```

Comment:
Oh oke with these changes the build fails again. Did not think about it.

Reply:
reverted to ckb again.